src/main/main.c

```diff
@@ -0,0 +1,24 @@
+/*
+ * This file is part of Betaflight-ESC.
+ *
+ * Betaflight-ESC is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Betaflight-ESC is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Betaflight-ESC.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+int main(void)
+{
+    return 0;

```

Comment:
Does this do Dshot1200? ;-P

Reply:
Sure, it's implemented inside `stdint` :-P

Reply:
DSHOT2400! 👍